src/shared/dbgutil/elfreader.cpp

```diff
@@ -32,37 +35,207 @@
 static const char ElfMagic[] = { 0x7f, 'E', 'L', 'F', '\0' };
 #endif
 
-extern bool ElfReaderReadMemory(void* address, void* buffer, size_t size);
+#ifdef HOST_UNIX
```

Comment:
That would probably be clearer.  We get away with HOST_UNIX because elfreader.cpp isn't included on OSX in CMakeLists.txt (just Linux and Windows). I'll change it.

Reply:
I just released that FREEBSD and NETBSD are considered UNIX so I think it needs to stay HOST_UNIX.